code/web/.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "args": [
+        "-u",
+        "tdd",
+        "--timeout",
+        "999999",
+        "--colors",
+        "${workspaceFolder}/test"
+      ],
+      "internalConsoleOptions": "openOnSessionStart",
+      "name": "Mocha Tests",
+      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+      "request": "launch",
```

Comment:
What are these lines doing?  Is it setting up the testing environment

Reply:
I forgot to remove this file, I thought I had captured it in the .gitignore. It was supposed to set up testing within the VSCode debugger, but it did not work, unfortunately.